travis-ci/managers/ubuntu.sh

```diff
@@ -19,5 +19,5 @@ if ! ldd build/libbpf.so | grep -q libelf; then
     echo "FAIL: No reference to libelf.so in libbpf.so!"
     exit 1
 fi
-make -j$((4*$(nproc))) -C src OBJDIR=../build DESTDIR=../install install
-rm -rf build install
+make -j$((4*$(nproc))) -C src V=1 OBJDIR=../build DESTDIR=../install install
```

Comment:
do you still need V=1 or it's just debug leftover?

Reply:
removed.